### ISSUE TITLE
Cannot run playbook with Python 3.

### ISSUE THREAD

**Issue by live2son** [moira-alert/moira-trigger-role#12]
python version - 3.7.3
ansible - 2.8.2

Playbook example:
```
ServiceNameTriggers:

  - id: ProjectServiceNameLoadAverage
    name: Project.ServiceName.LoadAverage
    targets:
      - Project.ServiceName.*.loadavg.load_normalized
    expression: "t1 > 1 ? ERROR : (t1 > 0.8 ? WARN : OK)"
    tags:
      - Project
      - DevOps
    desc: Символы отличные от латиницы
```

Playbook fails with:
```
"meta": {"failed": {"details": "'str' object has no attribute 'decode'", "error": "AttributeError", "method": "edit"}}, "msg": "Unable to define trigger state"}
```

**Comment by zhelyabuzhsky** [comment]
I have the same issue.

Python 3.7.5
Ansible 2.9.1

**Comment by zhelyabuzhsky** [comment]
I think it's fixed with #13
